gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/amqp/task/TaskDefinition.java

```diff
@@ -0,0 +1,13 @@
+package uk.nhs.adaptors.pss.translator.amqp.task;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.experimental.SuperBuilder;
+import lombok.extern.jackson.Jacksonized;
+
+@Jacksonized
+@SuperBuilder
+@Getter
+public class TaskDefinition {
```

Comment:
what about getting rid of that class completely? maybe we could add it when we need it?